deletefb/tools/wall.py

```diff
@@ -44,7 +44,10 @@ def delete_posts(driver,
                 try:
                     delete_button = menu.find_element_by_xpath("//a[@data-feed-option-name=\"FeedDeleteOption\"]")
                 except SELENIUM_EXCEPTIONS:
-                    delete_button = menu.find_element_by_xpath("//a[@data-feed-option-name=\"HIDE_FROM_TIMELINE\"]")
+                    try:
+                        delete_button = menu.find_element_by_xpath("//a[@data-feed-option-name=\"HIDE_FROM_TIMELINE\"]")
+                    except SELENIUM_EXCEPTIONS:
```

Comment:
I'll test this when I get home from work today, I believe this should do the trick.

That being said, I wonder if it makes more sense to find all possible `<a data-feed-option-name>` elements at once in a list, and then check loop over their option names in precendence of DELETE, HIDE, UNTAG, along with any other tags you may discover.

Reply:
Yes that makes sense. I wanted to make sure it didn't break anything first, but I think it should be ok to just check them in a loop and then the first one that actually exists in priority.